Controller/Block/Index.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
mogao si bez ovog file-a preko ajaxa, samo submit na controller koji pravi bookmark i ostalo riješit u JS-u ovisno koji response dobiješ tako ispisat što se desilo i nastavit dalje

Reply:
prepravio